src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -1437,19 +1437,32 @@ namespace
       : public RenderingManager::IScheduledAction
     {
       public:
+        explicit ClearRenderRegionAction(const AttributeEditor* attribute_editor)
+          : m_attribute_editor(attribute_editor)
+        {
+        }
+
         virtual void operator()(
             Project&        project) APPLESEED_OVERRIDE
         {
             project.get_frame()->reset_crop_window();
+            m_attribute_editor->refresh(
+                project.get_frame()->get_parameters());
         }
+
+      private:
+        const AttributeEditor* m_attribute_editor;
     };
 
     class SetRenderRegionAction
       : public RenderingManager::IScheduledAction
     {
       public:
-        explicit SetRenderRegionAction(const QRect& rect)
-          : m_rect(rect)
+        explicit SetRenderRegionAction(
```

Comment:
The `explicit` keyword is no longer required here.

I wish C++ had made constructors explicit by default...